specification/hdinsight/resource-manager/Microsoft.HDInsight/stable/2018-06-01-preview/cluster.json

```diff
@@ -1733,7 +1790,7 @@
       },
       "description": "Gateway settings."
     },
-    "OperationResource": {
+    "AsyncOperationResult": {
```

Comment:
Do we really need to rename this model? This renaming causes SDK breaking changes.
This model is not used at all in the previous version of this swagger, therefore could we just keep this name unchanged?

Reply:
> Do we really need to rename this model? This renaming causes SDK breaking changes.
> This model is not used at all in the previous version of this swagger, therefore could we just keep this name unchanged?

Hi @ArcturusZhang , Thanks for your comment.
 I think the new name is more meaningful. As you said, the class "OperationResource" is never used. I think this breaking change is ok, no customer will be impact.

I know this PR will cause breaking change but we have to fix S360 issues.I just submitted the sdk release requirement about one week ago.  Now we do **not** have the requirement for release sdk based this PR.  Thank you very much~

Reply:
Hi @ArcturusZhang , Is the reply accpetable for you? Besides this class name, there are other changes will cause breaking change too. This PR is fixing the S360 issues which will cause breaking changes in Sdks level.